src/database/db_connection.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Ruchita Nathani <ruchita.nathani@fau.de>
+
+import pymongo
+
+_client = None
+
+
+def mongo_connection(collection_name="default"):
+    global _client
+    if _client is None:
+        _client = pymongo.MongoClient("mongodb://root:example@mongodb:27017/")
```

Comment:
Here we should also use the environment variable you now created.

Reply:
Connection string Added in env. Please check now.